applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml

```diff
@@ -6,7 +6,9 @@ description: |-
     To enable the kubelet to rotate server certificates, edit the kubelet configuration
     file <tt>/etc/origin/node/node-config.yaml</tt>
     on the kubelet node(s) and set the below parameter:
```

Comment:
```suggestion
    on the kubelet node(s) and ensure <tt>RotateKubeletServerCertificate</tt> is included
    in the <tt>feature-gates</tt> arguments:
```

Reply:
@redhatrises how about this?

Reply:
That would work. Although we lose showing how to set it in the description.

Reply:
Not following how that'd be lost =/

Reply:
I reread. I was mistaken on it.

Reply:
@shawndwells updated.